x-pack/plugins/transform/public/app/sections/create_transform/components/step_details/step_details_form.tsx

```diff
@@ -48,11 +61,15 @@ export interface StepDetailsExposedState {
   touched: boolean;
   transformId: TransformId;
   transformDescription: string;
+  transformFrequency: string;
+  transformSettingsMaxPageSearchSize: number;
   valid: boolean;
-  indexPatternDateField?: string | undefined;
+  indexPatternTimeField?: string | undefined;
 }
 
 const defaultContinuousModeDelay = '60s';
+const defaultTransformFrequency = '1m';
```

Comment:
These settings don't seem to be copied over when cloning a job but reset to the defaults. Also even though it isn't exposed in the Create wizard, I think we should also copy over any `docs_per_second` setting.

Reply:
Good catch, added this to the issue about fixing other missing attributes when cloning transforms: https://github.com/elastic/kibana/issues/75264